.azuredevops/components/rocWMMA.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         -DCMAKE_BUILD_TYPE=Release
         -DROCWMMA_BUILD_TESTS=ON
         -DROCWMMA_BUILD_SAMPLES=OFF
-        -DGPU_TARGETS=gfx1100
+        -DAMDGPU_TARGETS=gfx942
```

Comment:
does this work with the different flag name?

Reply:
Their github readme uses AMDGPU_TARGETS: https://github.com/ROCm/rocWMMA
It was either a recent change or we have been using the wrong name for setting gpu targets

Reply:
The Cmakelists.txt does use AMDGPU_TARGETS, line 111: https://github.com/ROCm/rocWMMA/blob/develop/CMakeLists.txt#L111